src/main/resources/templates/request_reservation.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Booked</title>
+</head>
+
+<body>
+
```

Comment:
you could add a title here, something like `<h2> Reservation is confirmed </h2>`
Not necessary but could provide context.